docs/reference/intro.asciidoc

```diff
@@ -129,7 +158,7 @@ A simple {es} document might look like this:
 ==== Data and metadata
 
 An indexed document contains data and metadata.
-In {es}, metadata fields are prefixed with an underscore.
+In {es}, <<mapping-fields,metadata fields>> are prefixed with an underscore.
 
 The most important metadata fields are:
```

Comment:
yeah agree the goal was just to say here are some basics about what an elasticsearch document looks like if you have never seen one before, but _source is not super imp  

> consider dropping the list of most important metadata fields to avoid finger on the scale

💯